doc/source_tech_preview/contributor.rst

```diff
@@ -0,0 +1,243 @@
+.. _developerdocumentation:
+
+=======================
+Developer Documentation
+=======================
+
+In this section, you will find documentation relevant to developing
+socok8s.
+
+.. _contributing:
+
+Contributor Guidelines
+======================
+
+Submodules
+----------
+
+This repository uses submodules. All of the following guidelines apply
+to this socok8s repository only. Please check other projects' practices
+before contributing to them, if a change needs to happen there.
+
+Before submitting code
+----------------------
+
+This is a very fast moving project, and you should probably contact us
+before starting to work on it.
+
+If you're willing to submit code, please remember the following rules:
+
+* All code should match our
+  :ref:`codeguidelines`.
+* All code requires to go through our :ref:`reviews`.
+* Documentation should be provided with the
+  code directly. See also :ref:`documentation`.
+
+.. _bug_reporting:
+
+Bug reporting process
+---------------------
+
+Bugs should be filed as Github issues.
```

Comment:
That's not actually true - we want to leverage bugzilla for the public beta program for this.

Reply:
The program meeting decided not to let the tech preview users to file bugs directly, but send issues to the Beta mailing list. Is that aligned to what you expect?